src/components/transactions/TransactionList.vue

```diff
@@ -1,17 +1,55 @@
 <template>
   <div>
-    <TransactionItem
-      v-for="msg in transactions"
-      :key="msg.key"
-      :transaction="msg"
-      :validators="validators"
-      :address="address"
-    />
+    <template v-for="group in groupedTransactions">
+      <div :key="group[0].title">
+        <h1>{{ group[0].title }}</h1>
+        <TransactionItem
+          v-for="{ tx } in group"
+          :key="tx.key"
+          :transaction="tx"
+          :validators="validators"
+          :address="address"
+        />
+      </div>
+    </template>
   </div>
 </template>
 
 <script>
 import TransactionItem from "./TransactionItem"
+import groupBy from "lodash.groupby"
+import orderBy from "lodash.orderby"
+import moment from "moment"
+
+function stripTime(momentTime) {
+  return momentTime.startOf("day") // sets the time to 12AM
```

Comment:
Could we do without this?

Compare dates using `inBetween` and the granularity?

Reply:
Ah, ok I wasn't aware of that. Will change it.